crucible-llvm/src/Lang/Crucible/LLVM/MemModel/Partial.hs

```diff
@@ -262,16 +263,32 @@ annotateME sym mop rsn p =
        (BBMemoryError (MemoryError mop rsn))
      return p'
 
--- | Assert that the given LLVM pointer value is actually a raw bitvector and extract its value.
-projectLLVM_bv ::
+-- | Assert that the given LLVM pointer value is actually a raw bitvector and
+-- extract its value.
+ptrToBv ::
```

Comment:
I wonder if we should include a disclaimer that in general, you should prefer tagging your pointers with the relevant `UndefinedBehavior` check (e.g., `ComparePointerToBV`) rather than circumventing it with `ptrToBv`. (I think there are scenarios where `ptrToBv` is unavoidable, of course.)